src/store/index.js

```diff
@@ -8,18 +8,33 @@ Vue.use(Vuex);
 const state = {
   API_BASE: process.env.VUE_APP_API_BASE_URL,
   shouldDisplay: NotificationData.shouldDisplay,
+  publishCount: NotificationData.publishCount,
 };
 
+
 const mutations = {
   hideNotificationBanner() {
     localStorage.setItem('shouldDisplay', false);
     state.shouldDisplay = false;
+    localStorage.setItem('publishCount', NotificationData.publishCount);
   },
   initializeStore() {
     if (localStorage.getItem('shouldDisplay')) {
       state.shouldDisplay = false;
     }
   },
+  showNotificationBanner(){
+    if (localStorage.getItem('publishCount') < NotificationData.publishCount){
+        state.shouldDisplay = true;
+        localStorage.setItem('shouldDisplay', true);   
+    }
+    if(localStorage.getItem('publishCount') > NotificationData.publishCount){
+      //This is to catch an edge case if the users localstorage somehow becomes higher than the value we provide
+      localStorage.setItem('publishCount', NotificationData.publishCount)
+      state.shouldDisplay = true;
+      localStorage.setItem('shouldDisplay', true)
+    }
```

Comment:
Lines 28-29 is the same as 33-34 and the function's name is `showNotificationBanner` and I would suggest lines 1-2 below goes first. Then the extra edge case at the end. 
so lines 27-35 would now look like this:
```
1. state.shouldDisplay = true;   
2. localStorage.setItem('shouldDisplay', true);

if(localStorage.getItem('publishCount') &gt; NotificationData.publishCount){     
    //This is to catch an edge case if the users localstorage somehow becomes higher than the value we provide      
    localStorage.setItem('publishCount', NotificationData.publishCount)
}
```

Reply:
@Nick-P-Orr implemented a better solution that the one suggested; resolving feedback.